normandy/schema-check/test_api.py

```diff
@@ -0,0 +1,285 @@
+import configparser
+import pytest
+import requests
+
+
+@pytest.fixture
+def conf():
+    config = configparser.ConfigParser()
+    config.read('manifest.ini')
+    return config
+
+
+def test_recipes(conf, env):
+    r = requests.get(conf.get(env, 'api_root') + '/recipe')
+    response = r.json()
+    expected_fields = [
+        'id',
+        'last_updated',
+        'name',
+        'enabled',
+        'revision_id',
+        'action',
+        'arguments',
+        'filter_expression',
+        'current_approval_request',
+        'approval',
+        'is_approved'
+    ]
+
+    expected_arguments = [
+        'defaults',
+        'surveyId',
+        'surveys'
+    ]
+
+    expected_survey_fields = [
+        'message',
+        'thanksMessage',
+        'engagementButtonLabel',
+        'title',
+        'postAnswerUrl',
+        'weight',
+        'learnMoreUrl',
+        'learnMoreMessage',
+    ]
+
+    expected_default_fields = [
+        'message',
+        'thanksMessage',
+        'engagementButtonLabel',
+        'postAnswerUrl',
+        'learnMoreUrl',
+        'learnMoreMessage'
+    ]
+
+    # Verify we have at least one response
+    assert len(response) >= 1
```

Comment:
Is having no recipes whatsoever considered a failure state? From the app's perspective it's not, and at least from my standpoint having no recipes isn't necessarily wrong.